components/Ripple/src/MDCStatefulRippleView.m

```diff
@@ -99,6 +108,7 @@ - (void)updateActiveRippleColor {
 
 - (void)setRippleColor:(UIColor *)rippleColor forState:(MDCRippleState)state {
   _rippleColors[@(state)] = rippleColor;
+  [_rippleDefaultColors removeAllObjects];
```

Comment:
So if I understand this logic correctly, once I override the value for `.normal` state, I lose all the defaults for the `.selected` and `.dragged` states?   This seems not great, since I'd expect my changes to only affect (1) the `.normal` state or (2) any state that was previously "falling back" to `.normal`.